### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by lnrego**
Add Portuguese README.md file

### CODE REVIEW

**Comment by lnrego** [comment]
You mean the, "Read this in other languages: 한국어、中国." or there are another things needed?

@stevemart

**Comment by stevemart** [comment]
@lnrego take a look at how it is rendered: https://github.com/lnrego/personal-wealth-portfolio-mgt-bot/blob/308bf138d534b8d08c4eb167fb8e5895d81e9f9e/README-pt.md

For instance, the titles need new lines between the titles and content

**Comment by lnrego** [comment]
@stevemart 

All fixes done, those md were not done by me, now it's all OK.

File: README-pt.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+  * Em caso de execução local, inspecione as variáveis de ambiente com atenção para confirmar se correspondem.
+
+  As credenciais para serviços do Bluemix (Conversation, Cloudant e Discovery) podem ser encontradas no menu ``Services``, no Bluemix, ao selecionar a opção ``Service Credentials``.
+
+
+  * Uma alternativa é depurar o aplicativo acessando `https://<name of your application>.mybluemix.net/debug.html` para ver um painel que exibe metadados que contêm detalhes sobre a interação com os serviços que estão sendo usados.
+
+# Licença
+
+[Apache 2.0](LICENÇA)
```
Comment (stevemart) [inline README-pt.md]: hmm, this should still be "LICENSE' since it was a link to a file

**Pull request merged**